### PR TITLE
allow black skybox

### PR DESCRIPTION
**Pull request by zzmp**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: ca92e5d161a12cef3d6f2fdf7a5b85087b7b8790 available at: https://gist.github.com/9d76ea9f8708840eda8abdf0383ade84<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8535-ca92e5d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8535-ca92e5d.dmg

**Comment by zzmp** [comment]
QA:
Create a zone and get in it. Ensure the following:
- Background mode set to `Nothing`: default background is visible (blue horizon)
- Background mode set to `Skybox`:
  - No `Skybox URL` set: skybox is a solid color, as specified. Do not use a color checker, we do tone-mapping based on lighting, so it may not be exactly as specified.
  - `Skybox URL` set, no color set: skybox is the specified texture.
  - `Skybox URL` set, color set: skybox is the specified texture, tinted by the color. Note that Black(0,0,0) has no value, and will not tint it at all.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5f699072d9c101874a0de1526be617d2d06e6ad0 available at: https://gist.github.com/9d76ea9f8708840eda8abdf0383ade84<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8535-5f69907.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8535-5f69907.dmg

**Comment by OmegaHeron** [comment]
QA Pass

Testing Windows 10

Moved to 10000,10000,10000 to get well away from standard home scene.
Created zone and insured I'm inside it.
Background mode = nothing.  I see blue horizon.
Background mode = skybox.  Skybox URL = blank.  Defaults to blackness.  Changing color results in a change matching expected.
Background mode = skybox.  Skybox URL = sunsset sky Color = 0,0,0.  Appears as expected.
Same as above, but, varying coloration.  Skybox texture takes on tint from color as expected.

Sanity checks.  Returning to standard home content shows zone as normal.  Going to other sims I'm familiar with shows all zones as expected.

File: libraries/model/src/model/Skybox.cpp [hunk 1]
```diff
@@ -50,6 +52,7 @@ void Skybox::updateSchemaBuffer() const {
 }
 
 void Skybox::clear() {
+    _empty = true;
```
Comment (zzmp) [inline libraries/model/src/model/Skybox.cpp]: This is a bug, `setCubemap` will revert it to `false`.
Found it a few minutes ago debugging your feedback on #8530.
Will fold the fix into #8530.

**Pull request merged**